electrumx/server/session.py

```diff
@@ -2990,6 +2988,9 @@ async def get_transaction_detail(self, txid, height=None, tx_num=-1):
 
         if res.get("op"):
             self.session_mgr._tx_detail_cache[tx_hash] = res
+
+        # Recursively encode payload content.
+        res["info"]["payload"] = auto_encode_bytes_elements(res["info"]["payload"])
```

Comment:
If there is relevant 'op' in the returned res, the corresponding payload information will be added to the returned structure. Here may be a KeyError issue.

Reply:
Only mint op have payload

Reply:
Here you go. Now it encodes the whole result dict. :)